runtime/moonbase/tests/common/mod.rs

```diff
@@ -0,0 +1,259 @@
+// Copyright 2019-2021 PureStake Inc.
```

Comment:
if there are no other files in the `common` folder, why not just make this a file `common.rs` and the imports in the other files would be the same?

There's no real difference, I just don't see the benefit of using a folder instead of a file if the folder doesn't contain anything else but a single file.

Reply:
Actually, I think it'd be more appropriate for this to be the `tests/mod.rs` and then we can declare the other files as mods and `use super::*` in each of them.

Reply:
In both cases - common.rs and mod.rs - Cargo will [compile them as a separate crate](https://doc.rust-lang.org/book/ch11-03-test-organization.html#integration-tests) and, additionally, `common` or `mod` will appear in the test result with `running 0 tests`. 

To avoid that, I just created the module using a folder instead, as [detailed here](https://doc.rust-lang.org/book/ch11-03-test-organization.html#submodules-in-integration-tests).

> To avoid having common appear in the test output, instead of creating tests/common.rs, we’ll create tests/common/mod.rs. This is an alternate naming convention that Rust also understands. Naming the file this way tells Rust not to treat the common module as an integration test file.

Reply:
Cool, I will set this conversation to unresolved so others will see it, but it is resolved.